superbench/benchmarks/model_benchmarks/pytorch_cnn.py

```diff
@@ -14,6 +14,12 @@
 from superbench.benchmarks.model_benchmarks.pytorch_base import PytorchBase
 from superbench.benchmarks.model_benchmarks.random_dataset import TorchRandomDataset
 
+def _keep_BatchNorm_as_float(module):
```

Comment:
could you help to fix the lint issue? expected 2 blank lines before and after this function, you can check by `python3 setup.py lint`.